tests/collections/nlp/test_prompt_learning.py

```diff
@@ -83,6 +83,15 @@ def get_task_templates():
     return task_templates
 
 
+def get_pseudo_tokens(total_virtual_tokens):
+    pseudo_tokens = [
+        VirtualPromptPlaceholderToken.BASE.value + str(i) + VirtualPromptPlaceholderToken.END.value
+        for i in range(total_virtual_tokens)
+    ]
+
+    return pseudo_tokens
+
```

Comment:
I see you have the similar logics in the prompt_lenaring_model file. Put the function there?

Reply:
Sure, I can do that